### ISSUE TITLE
Feature request: Move "Update All" button away from "Next Unread Item"

### ISSUE THREAD

**Issue by Ottaviocr** [lwindolf/liferea#1060]
Hi,

`$ liferea -v
Liferea 1.12.6
$ uname -a
Linux e130 4.19.0-17-amd64 #1 SMP Debian 4.19.194-3 (2021-07-18) x86_64 GNU/Linux
`
Maybe it,s just me, but I often end up clicking on the "Update All" button whereas I wanted to click on "Next Unread Item". I am aware this is my fault, but I wonder if it wouldn't be better to move these buttons away from each other and avoid triggering unwanted downloads. Thanks.

**Comment by rambit** [comment]
I agree, and I also did it a few times. I don't know if it adds to the problem that the pane is not actually visual buttons before you move your mouse over them. As I also wrote elsewhere, the Update All button should be disabled while it's actually updating.

But I'm writing because an alternative solution to moving it could be an "are you sure?" dialog when you have unread feeds, or based on how long since last time you clicked it.

**Comment by Ottaviocr** [comment]
+1 for the "Are you sure" as a compromise solution.

-- 
Ottavio Caruso

A: Because it messes up the order in which people normally read text.
Q: Why is top-posting such a bad thing?
A: Top-posting.
Q: What is the most annoying thing in e-mail?

**Comment by sjehuda** [comment]
> +1 for the "Are you sure" as a compromise solution.

+1 too

**Comment by lwindolf** [comment]
With the 2.0 headerbar implementation I plan to split the headerbar between feed list and other panes, this will split the buttons to isolating "Update All" from "Next Unread"
